### ISSUE TITLE
Vtop gets crazy as soon as I move the mouse

### ISSUE THREAD

**Issue by Darkhogg** [MrRio/vtop#57]
As soon as I start moving my mouse, vtop quits (probably crashes) and I start to see what looks like the mouse coordinates in ASCII in the terminal input. It sometimes comes back to normal after a few moments, others I have to run `reset`. This happens on both `terminator` and `xfce4-terminal`. I haven't tried anything else.

**Comment by Darkhogg** [comment]
[Tim Aniol](https://plus.google.com/100888634760270100642) on [a Google+ post](https://plus.google.com/114339166306139301175/posts/W1bbRBSMZHZ) claims to have seen this in urxvt, at least the closing issue.

**Comment by d-oliveros** [comment]
I happens on recent versions of Node. I'm using 0.10.36 and it works correctly, but breaks when using 0.12.0 (or io.js)

**Comment by d-oliveros** [comment]
Hey @Darkhogg, this PR fixes the issue. The problem is an outdated package dependency (probably blessed or drawille). After updating the packages to the most recent versions, it works flawlessly. @MrRio thoughts?

**Comment by Darkhogg** [comment]
@d-oliveros I cloned your repo to try it and it looks like it works now, it doesn't get closed.
I does however add some garbage to my input, as if I wrote it, in particular my prompt had this after doing some random mouse moving:

    C<+C=+C>+C?+C@+CA*CB*CD*

I suppose there were some ESC chars, too, but were invisible. Anyway, vtop works now for me, this is just an annoyance, and seeing that the problem were other packages, I personally consider this fixed. Let's see what the author thinks.
